### PR TITLE
Readd Death Ripley's clamps for scott only.

### PR DESCRIPTION
**Pull request by SarmentiCampbell**
## About The Pull Request
Scott has a tendency to try and kill himself, with this pr, we can finally pacify him by rendering him harmless

## Why It's Good For The Game
It's hrp, trust me

## Changelog
:cl:
/:cl:

### CODE REVIEW

File: code/modules/vehicles/mecha/equipment/tools/work_tools.dm [hunk 1]
```diff
@@ -107,7 +107,7 @@
 				"<span class='notice'>[chassis] pushes you aside.</span>")
 			return ..()
 
-		/*else if(LAZYACCESS(modifiers, RIGHT_CLICK) && iscarbon(M))//meme clamp here
+		else if(LAZYACCESS(modifiers, RIGHT_CLICK) && iscarbon(M) && M.real_name == "Scott Waverly")//meme clamp here //SKYRAT EDIT - Original : else if(LAZYACCESS(modifiers, RIGHT_CLICK) && iscarbon(M))
```
Comment (tf-4) [inline code/modules/vehicles/mecha/equipment/tools/work_tools.dm:110]: downvoted

**Review by tf-4 — CHANGES_REQUESTED** [review]
bad change

**Review by RatFromTheJungle — APPROVED** [review]
LGTM

File: code/modules/vehicles/mecha/equipment/tools/work_tools.dm (hunk 1 above)
Comment (SarmentiCampbell) [reply code/modules/vehicles/mecha/equipment/tools/work_tools.dm:110]: :)

**Comment by NotRanged** [comment]
Good PR

**Comment by Gandalf2k15** [comment]
no memes ngtm

**Pull request closed without merging**